### PR TITLE
Update DaemonSet scheduling version inconsistency

### LINKED ISSUE #17336: Issue with k8s.io/docs/concepts/workloads/controllers/daemonset/
**This is a Bug Report**

**Problem:**
"A DaemonSet ensures that all eligible nodes run a copy of a Pod. Normally, the node that a Pod runs on is selected by the Kubernetes scheduler. However, DaemonSet pods are created and scheduled by the DaemonSet controller instead."

This is shown as the default option for a daemonset, but just above, it says the opposite is the default behavior.

**Proposed Solution:**
Change the default section or change this paragraph to show the correct behavior

**Page to Update:**
https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/

### PR DESCRIPTION
**Pull request by christophercarney**
Fixes #17336

Check the issue for more details but in short: an update was made to the header specifying the default behavior when it switched over in 1.12 without modifying the intro paragraph (which still reflects that DaemonSet controller is the default scheduling behavior)

### CODE REVIEW

File: content/en/docs/concepts/workloads/controllers/daemonset.md [hunk 1]
```diff
@@ -117,9 +117,9 @@ when the Pod is created, so it is ignored by the scheduler).  Therefore:
 
 {{< feature-state state="beta" for-kubernetes-version="1.12" >}}
 
-A DaemonSet ensures that all eligible nodes run a copy of a Pod. Normally, the
+A DaemonSet resource type ensures that all eligible nodes run a copy of a Pod. Normally, the
 node that a Pod runs on is selected by the Kubernetes scheduler. However,
-DaemonSet pods are created and scheduled by the DaemonSet controller instead.
+DaemonSet pods can also be created and scheduled by a DaemonSet controller instead.
```
Comment (sftim) [inline content/en/docs/concepts/workloads/controllers/daemonset.md]: How about:
```suggestion
The DaemonSet controller can also schedule the DaemonSet pods that it creates.
```
?

File: content/en/docs/concepts/workloads/controllers/daemonset.md [hunk 2]
```diff
@@ -117,9 +117,9 @@ when the Pod is created, so it is ignored by the scheduler).  Therefore:
 
 {{< feature-state state="beta" for-kubernetes-version="1.12" >}}
 
-A DaemonSet ensures that all eligible nodes run a copy of a Pod. Normally, the
+A DaemonSet resource type ensures that all eligible nodes run a copy of a Pod. Normally, the
```
Comment (sftim) [inline content/en/docs/concepts/workloads/controllers/daemonset.md]: Why this change? Because DaemonSet is written with titlecase it looks like a type of Kubernetes object, and it is.

File: content/en/docs/concepts/workloads/controllers/daemonset.md [hunk 3]
```diff
@@ -117,9 +117,9 @@ when the Pod is created, so it is ignored by the scheduler).  Therefore:
...
 node that a Pod runs on is selected by the Kubernetes scheduler. However,
-DaemonSet pods are created and scheduled by the DaemonSet controller instead.
+DaemonSet pods can also be created and scheduled by a DaemonSet controller instead.
 That introduces the following issues:
```
Comment (sftim) [inline content/en/docs/concepts/workloads/controllers/daemonset.md]: This text reads like it's describing a new feature, rather than something that has been the default for several releases.
`ScheduleDaemonSetPods` is the default behavior and, even though it's in beta, to me it makes sense to document the older behavior as the exceptional one.

@christophercarney do you agree, and if so do you want to cover that clean-up in this PR?

Comment (christophercarney) [reply content/en/docs/concepts/workloads/controllers/daemonset.md]: I agree and that was something that occurred to me as I was writing so I'm glad you pointed that out. I will make that change in the scope of this PR.

File: content/en/docs/concepts/workloads/controllers/daemonset.md (hunk 2 above)
Comment (christophercarney) [reply content/en/docs/concepts/workloads/controllers/daemonset.md]: Since the previous section only had references to DaemonSet controller, I wanted to be explicit and re-center the conversation to make it clear we were now once again talking about the DaemonSet resource type rather than just the controller, let me know if you prefer it without

Comment (sftim) [reply content/en/docs/concepts/workloads/controllers/daemonset.md]: Previously Kubernetes talked about “ReplicationController” etc, as if the control loop were more important than the resource. Things are moving to talk about ReplicaSet, Deployment, DaemonSet etc rather than “a ReplicaSet controller”. The main exception to that is if you are actually documenting the specifics of how those control loops are implemented.

**Comment by sftim** [comment]
@christophercarney you might also have some thoughts on rewording https://github.com/kubernetes/website/pull/16885/files#diff-8342a130ee45184367af9dca4b6d6f2aR1 
If you do, please feel free to make those suggestions!

**Comment by christophercarney** [comment]
@sftim Made a few changes according to our discussion. Kept the rewording but moved the motivation for why DaemonSet controller is no longer the default behavior to the DaemonSet controller section to (hopefully) make it more clear both why default scheduler is the default behavior and that it _is_ indeed the default.

**Comment by christophercarney** [comment]
I’ll take note of that and circle back around to this at that time.

/assign @tfogo

**Comment by sftim** [comment]
This didn't get approved as yet.
@christophercarney are you willing to revise this now that v1.17 is out?

**Comment by kbhawkey** [comment]
Hello @christophercarney , Do you have time to follow-up on this PR (and rebase as well)?

**Comment by sftim** [comment]
@christophercarney I'm going to close this PR. _Please feel free_ to rebase your work against _master_ and open a new similar pull request with the same description.

/close
